src/commands/docs/providers/baseline.ts

```diff
@@ -0,0 +1,125 @@
+import {
+  ActionRowBuilder,
+  ButtonBuilder,
+  ButtonStyle,
+  Collection,
+  EmbedBuilder,
+  type MessageActionRowComponentBuilder,
+  StringSelectMenuBuilder,
+} from 'discord.js';
+import { features as data } from 'web-features';
+import { fuzzySearch } from '../../../util/fuzzy-search.js';
+import type { ProviderConfig } from '../types.js';
+import { createBaseConfig } from '../utils.js';
+
+export type FeatureData = {
+  name: string;
+  description: string;
+  status: {
+    baseline: 'high' | 'low' | false;
+    support: Record<string, string>;
+  };
+};
+
+const features = Object.fromEntries(
+  Object.entries(data).filter(([, feature]) => feature.kind === 'feature')
+) as Record<string, FeatureData>;
```

Comment:
I logged the original object and noticed, that only two don't have feature.
```suggestion
const features = { ...data };
delete features['numeric-seperators'];
delete features['single-color-gradients'];
```
To prevent future deploments to run into an issue where a new non-feature is added, I suggest writing a simple unit test that checks that only the non-features exist we expect.

For example inside utils.ts you can have
```ts
fxport const NON_FEATURES = ['numeric-seperators', 'single-color-gradien'];

export function getFeaturesOnly(originalFeatures) {
  const features = { ...data };
  NON_FEATURES.forEach(nonFeature => {
    delete features[nonFeature];
  })

  return features;
}
```
and in `baseline.ts` just do `const features = getFeaturesOnly(data);`
and then in `utils.test.ts` filter by non features and check that every non-feature is in our NON_FEATURE const.